### PR TITLE
Add rubygem-hammer_cli_foreman_ansible package

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

File: packages/plugins/rubygem-hammer_cli_foreman_ansible/rubygem-hammer_cli_foreman_ansible.spec [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+%config %{hammer_confdir}/cli.modules.d/%{plugin_name}.yml
+
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/README.md
+%doc %{gem_instdir}/config
+%{gem_instdir}/test
+
+%changelog
+* Fri Apr 13 2018 Oleh Fedorenko <ofedoren@redhat.com> 0.1.0-1.fm1_18
```
Comment (ekohl) [inline packages/plugins/rubygem-hammer_cli_foreman_ansible/rubygem-hammer_cli_foreman_ansible.spec]: I don't think we usually include the dist tag here. If this was generated by our tools, it's probably a bug. Could you make this `0.1.0-1`?

File: packages/plugins/rubygem-hammer_cli_foreman_ansible/rubygem-hammer_cli_foreman_ansible.spec [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.0
+Release: 1%{?foremandist}%{?dist}
+Summary: Foreman Ansible plugin for Hammer CLI
+Group: Development/Languages
+License: GPL-3.0
+URL: https://github.com/theforeman/hammer-cli-foreman-ansible
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
```
Comment (ekohl) [inline packages/plugins/rubygem-hammer_cli_foreman_ansible/rubygem-hammer_cli_foreman_ansible.spec]: Could you remove the trailing whitespace here (and below)?

**Comment by ofedoren** [comment]
@ekohl done. And yes, everything was generated by the tools.

**Comment by ehelms** [comment]
[test rpm]

**Comment by ekohl** [comment]
[test rpm]

**Comment by xprazak2** [comment]
@ofedoren, could you take a look at the failures pls?

**Comment by ofedoren** [comment]
Apologize for taking so long, now it should be ok.

@ekohl updated.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**